server/utils/helpers/ratingControllerHelper.js

```diff
@@ -0,0 +1,37 @@
+import errorHelper from './errorHelper';
+
+/**
+ * Check if article exists, and check that the one rating
+ * is not the author
+ * @param {Object} article article object retrieved from the database
+ * @param {Number} userId user ID from the token
+ * @returns {Boolean} return true on success or throw an error
+ */
+const validateRating = (article, userId) => {
+  if (!article) {
+    return errorHelper.throwError('Article not found', 404);
+  }
+  if (article.authorId === userId) {
+    return errorHelper
+      .throwError('You can\'t provide ratings for your article', 403);
+  }
+  return true;
+};
+
+/**
```

Comment:
jsdoc ??

Reply:
These hound violations have been fixed, you can check the summary at the button of the PR, the comments are still there because that line hasn't changed.